### PR TITLE
[12.0] [MIG] analytic_base_department

### PR DESCRIPTION
**Pull request by anandkansagra**
Issue [Reference](https://github.com/OCA/account-analytic/issues/189)

### CODE REVIEW

**Review by emagdalenaC2i — APPROVED** [review]
Code review

File: analytic_base_department/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2011-2018 Camptocamp SA
+# Copyright 2013 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Base Analytic Department Categorization",
+    "version": "12.0.1.0.0",
+    "author": "Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
```
Comment (nikul-serpentcs) [inline analytic_base_department/__manifest__.py:7]: @anandkansagra `Daniel Reis`  name in an author.
@dreispt can you please send company name?

Comment (dreispt) [reply analytic_base_department/__manifest__.py:7]: There is no company name to go there.

File: analytic_base_department/__manifest__.py [hunk 2]
```diff
@@ -3,13 +3,12 @@
...
 {
     "name": "Base Analytic Department Categorization",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "author": "Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "category": "Generic Modules/Projects & Services",
     "website": "https://github.com/OCA/account-analytic",
     "depends": ["account",
-                "analytic",
```
Comment (dreispt) [inline analytic_base_department/__manifest__.py]: Why is this removed? The module uses `account.analytic.account` Model, provided by the `analytic` module.

File: analytic_base_department/views/analytic.xml [hunk 3]
```diff
@@ -1,70 +1,70 @@
...
+    <record id="view_account_analytic_account_form" model="ir.ui.view">
+        <field name="name">account.analytic.account.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
+        <field name="arch" type="xml">
+            <field name="code" position="after">
+                <field name="department_id"/>
+            </field>
+        </field>
+    </record>
```
Comment (dreispt) [inline analytic_base_department/views/analytic.xml:13]: No point in changing the XML indentation, but discussing that would be bikeshedding.

File: analytic_base_department/__manifest__.py (hunk 2 above)
Comment (anandkansagra) [reply analytic_base_department/__manifest__.py]: @dreispt ,

If we will install the `account` module then `analytic` module will automatically installed due to dependency. So no need to put `analytic` module in the dependency
Here is the Reference:- https://github.com/odoo/odoo/blob/12.0/addons/account/__manifest__.py#L18

**Review by dreispt — COMMENTED** [review]
Thanks @anandkansagra, good catch. I would then prefer to keep the closest dependency: this needs `analytic` but doesn't need `account`. So I would rather remove the `account` dependency.

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Comment by anandkansagra** [comment]
> Thanks @anandkansagra, good catch. I would then prefer to keep the closest dependency: this needs `analytic` but doesn't need `account`. So I would rather remove the `account` dependency.

@dreispt ,
Technically it will not throw an error. I have seen that in the `analytic` module, there seems no menuitem. All the menuitem related `analytic` module is defined in the `account` module. So do we need to change it in this PR?

**Comment by pedrobaeza** [comment]
I have squashed a bit commit history and merging when finished.
